packages/react/src/components/form/Radio/RadioListElement.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { InputHTMLAttributes, useContext, useMemo } from "react";
+import styled from "styled-components";
+import Text from "../../asorted/Text";
+import Flex, { FlexBoxProps } from "../../layout/Flex";
+import { RadioContext } from "./index";
+
+const Label = styled(Text)<{ checked: boolean; disabled: boolean | undefined }>`
+  color: ${(p) =>
+    p.disabled
+      ? p.theme.colors.palette.neutral.c50
+      : p.checked
+      ? p.theme.colors.palette.primary.c90
+      : p.theme.colors.palette.neutral.c100};
+`;
+
+const Container = styled(Flex)<{ checked: boolean; disabled: boolean | undefined }>`
+  cursor: ${(p) => (p.disabled ? "" : "pointer")};
```

Comment:
Not important at all, but good to know (personally I discovered that not long ago) 📚 : 

> [styled-components ignores interpolations that evaluate to undefined, null, false, or an empty string (""), which means you're free to use short-circuit evaluation to conditionally add CSS rules](https://styled-components.com/docs/advanced#tagged-template-literals)

Which means that you could for instance write the same line as `p => !p.disabled && "pointer"`